Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -308,28 +308,29 @@ def create_values_for_returned_dict(iocs: list, request_args: RequestArguments)
         # protocol stripping
         indicator = _PROTOCOL_REMOVAL.sub('', indicator)
 
-        # Port stripping
-        indicator_with_port = indicator
-        # remove port from indicator - from demisto.com:369/rest/of/path -> demisto.com/rest/of/path
-        indicator = _PORT_REMOVAL.sub(_URL_WITHOUT_PORT, indicator)
-        # check if removing the port changed something about the indicator
-        if indicator != indicator_with_port and not request_args.url_port_stripping:
-            # if port was in the indicator and url_port_stripping param not set - ignore the indicator
-            continue
-        # Reformatting to to PAN-OS URL format
-        with_invalid_tokens_indicator = indicator
-        # mix of text and wildcard in domain field handling
-        indicator = _INVALID_TOKEN_REMOVAL.sub('*', indicator)
-        # check if the indicator held invalid tokens
-        if with_invalid_tokens_indicator != indicator:
-            # invalid tokens in indicator- if drop_invalids is set - ignore the indicator
-            if request_args.drop_invalids:
+        if ioc_type not in ['IP', 'IPv6']:
```

Comment:
1. You can use the constants from CSP for it.
2. Do you need to add CIDRs as well?